### PR TITLE
dev-db/pgcli: Fix test dependencies

### PR DESCRIPTION
**Pull request by jsmolic**
Closes: https://bugs.gentoo.org/765622

### CODE REVIEW

File: dev-db/pgcli/pgcli-3.1.0.ebuild [hunk 1]
```diff
@@ -31,7 +31,6 @@ BDEPEND="
 	test? (
 		dev-db/postgresql
 		dev-python/mock[${PYTHON_USEDEP}]
-		dev-python/python-dateutil[${PYTHON_USEDEP}]
```
Comment (thesamesam) [inline dev-db/pgcli/pgcli-3.1.0.ebuild:34]: If it directly needs dateutil, keep it in

File: dev-python/pendulum/pendulum-2.1.2.ebuild [hunk 2]
```diff
@@ -21,7 +21,7 @@ KEYWORDS="~amd64 ~x86"
 # commented for future tests fixups
 RESTRICT="test"
 
-RDEPEND="
+DEPEND="
```
Comment (thesamesam) [inline dev-python/pendulum/pendulum-2.1.2.ebuild:24]: I'll revbump anyway because it's cheap for a Python package

Comment (jsmolic) [reply dev-python/pendulum/pendulum-2.1.2.ebuild:24]: Thanks, I forgot to revbump

File: dev-db/pgcli/pgcli-3.1.0.ebuild (hunk 1 above)
Comment (jsmolic) [reply dev-db/pgcli/pgcli-3.1.0.ebuild:34]: Actually `pendulum` needs it, but didn't pull it in due to DEPEND issue

**Pull request closed without merging**